manifests/2.8.0/opensearch-2.8.0.yml

```diff
@@ -76,18 +67,19 @@ components:
       - gradle:dependencies:opensearch.version
   - name: ml-commons
     repository: https://github.com/opensearch-project/ml-commons.git
-    ref: 2.x
+    ref: '2.8'
     platforms:
       - linux
       - windows
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version: opensearch-ml-plugin
-  - name: performance-analyzer
-    repository: https://github.com/opensearch-project/performance-analyzer.git
-    ref: 2.x
+  - name: neural-search
```

Comment:
I think you added `neural-search` twice at the bottom of this PR.

Reply:
Why is this check not working? https://github.com/opensearch-project/opensearch-build/blob/main/.yamllint.yml#L28

Reply:
Fixed, did `git push` again instead of `--amend`, please ignore that. @zelinh